### PR TITLE
Add iaas team-meeting-minutes from 2022.08.24

### PR DESCRIPTION
**Pull request by ra-beer**
Signed-off-by: Ramona Rautenberg <rautenberg@osism.tech>

### CODE REVIEW

File: iaas/20220824.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+* Anything else to add to the agenda?
+
+## Agenda
+
+* https://github.com/SovereignCloudStack/issues/issues/146
+* Upgrade path for R3
+
+## Automated upgrade tests yoga (#146)
+
+* to be checked wether this is done at osism (@ra-rau)
```
Comment (fkr) [inline iaas/20220824.md:25]: should then be corrected to @ra-beer

**Review by fkr — CHANGES_REQUESTED** [review]
see comment regarding your username ;)

**Comment by fkr** [comment]
Your DCO still says 'Ramona Rautenberg'.... @ra-beer

File: iaas/20220824.md (hunk 1 above)
Comment (ra-beer) [reply iaas/20220824.md:25]: ups yes.

**Comment by ra-beer** [comment]
Yes, i forget to change this. For the next PR it will be changed. But for this i will let it, so i dont get DCO errors.

**Review by fkr — APPROVED** [review]

**Comment by fkr** [comment]
LGTM. Thanks!

**Pull request merged**